src/main.ts

```diff
@@ -17,10 +17,13 @@ export async function run(): Promise<void> {
         console.log("No vulnerabilities were detected in the container image");
     } else {
         const errors = utils.extractErrorsFromLogs(trivyHelper.getTrivyLogPath(), trivyHelper.trivyToolName);
+        
+        console.log(errors);
+
         errors.forEach(err => {
             core.error(err);
         });
-        throw new Error("An error occurred while scanning the container image for vulnerabilities");
+        throw new Error("An error occurred while scanning container image: " + inputHelper.imageName + " for vulnerabilities. Please check your image name and try again.");
```

Comment:
> I think we should not add the text Please check your image name and try again. as the scan could fail for many reasons, like incorrect/expired auth, upstream error etc. Thoughts?

Agreed. Given that you're printing out the errors in the above step, it can be figured out.

Reply:
Resolved. Removed `Please check your image name and try again`.